src/main/scala/uk/gov/nationalarchives/omega/api/business/echo/EchoService.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2023 The National Archives
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package uk.gov.nationalarchives.omega.api.business.echo
+
+import cats.data.{ NonEmptyChain, Validated }
+import uk.gov.nationalarchives.omega.api.business.{ BusinessService, BusinessServiceError, BusinessServiceRequest, BusinessServiceResponse, RequestValidation }
+
+class EchoService extends BusinessService with RequestValidation {
+
+  override def validateRequest(request: BusinessServiceRequest): ValidationResult =
+    Validated.cond(request.text.nonEmpty, request, NonEmptyChain.one(TextIsNonEmptyCharacters))
+
+  override def process(request: BusinessServiceRequest): Either[BusinessServiceError, BusinessServiceResponse] =
+    if (request.text.contains("ERROR")) {
```

Comment:
Why do we have this?

Reply:
This is just to allow the error path to be triggered. In other words if you send a message that says "ERROR" you will get an error message back. This is not a validation error but an error that arises as a result of processing a valid request. The purpose is just to show how an error would be handled.

Reply:
So it's not code that is currently being used.

If that's the case, I'd keep it out until then. We have the original (the prototype) anyway.

Resolving anyway.